### DISCUSSION TITLE
Rotate camera

### DISCUSSION THREAD

**Discussion by Caouette1988** [mainsail-crew/spyglass#29] [Q&A] [answered]
Hi, ty for the nice work, my pi camera v3 work great now.

Is there a way to rotate camera 90deg, there is no option in mainsail when selecting V4L-MJPEG mode.

Thank you.

**Comment by roamingthings** [comment]
Currently spyglass does not offer a way to rotate the image.

**Reply by DenDovne** [reply]
isn't there an update from a couple of days ago that let you flip it vertically and horizontally? also, if there is, I cant figure it out for the life of me. I'm using mainsail os, and when I load (my IP)/webcam/stream, nothing I do, I cant seem to get the image to flip vertically. https://github.com/roamingthings/spyglass/pull/13  I don't know if I have to edit the cli.py file or the camera.py file. i tried to change it from false to true in the camera.py file but it didn't make a difference. not sure if id also need to do that and add to the spyglass.conf file.. I'm at a lost and any help would be greatly appreciated. thank you

**Reply by roamingthings** [reply]
There is an open pull request #37 that adds image rotation by adding EXIF to the images/stream. However, this pull request is not done yet and still needs some work.

**Comment by Caouette1988** [comment, marked as answer]
This question have been solved by the last update 0.11.0
